### DISCUSSION TITLE
Disable "Support banner" bottom of the page

### DISCUSSION THREAD

**Discussion by b-onc** [photoprism/photoprism#3740] [Q&A] [answered]
I've just started using Photoprism (photos still indexing) but the sticky "Support Photoprism" banner at the bottom of the web app is too distracting, especially on mobile. I'll definitely support Photoprism if I find it useful and I'm still trying it out.

Is there a way to disable this banner?

**Comment by lastzero** [comment, marked as answer]
Your version seems to be very old, so simply upgrade to a more recent release:

https://docs.photoprism.app/release-notes/

**Reply by b-onc** [reply]
Thank you! Updating was smooth. Unfortunately after updating, I've lost all my video thumbnails, and many more thumbnails. Do I have to re-index after each update? My library is around 500GB and reindexing on my raspi4 takes around 3 days.

**Reply by graciousgrey** [reply]
Re-indexing should not be needed after an update. Have you properly mounted your storage folder or does it only exist within the container?

https://docs.photoprism.app/getting-started/docker-compose/#photoprismstorage

**Reply by b-onc** [reply]
yes, it's properly mounted. anyways i wanted to change thumbnail sizes so i removed everything and will start from scratch. thanks for the answer.
